utils/build-script-impl

```diff
@@ -3124,8 +3124,11 @@ function build_and_test_installable_package() {
             with_pushd "${PKG_TESTS_SANDBOX_PARENT}" \
                 call tar xzf "${package_for_host}"
 
+            if python -c import psutil ; then
+              TIMEOUT_ARGS=--timeout=3000 # 50 minutes
```

Comment:
Can we change this to 20mins? Is there a reason for 50mins?

Reply:
The most technical reason ever: copy paste!